README.md

```diff
@@ -483,14 +541,89 @@ that can be used to submit SQL queries. The endpoint is accessible under
 SQL statements are sent as the body of POST requests in JSON format, where the
 SQL statement is the value of the `stmt` attribute.
 
-### Read Schemas
+**QuantumLeap** offfers an API wrapping CrateDB backend so you can also perform
+multiple types of queries directly talking to QuantumLeap. The documentation of
+the API is [here](https://app.swaggerhub.com/apis/smartsdk/ngsi-tsdb/). Mind the
+versions. If you have access to your quantumleap container (e.g. it is running
+in locahost or port-forwarding to it), you can navigate its API via 
+[http://localhosts:8668/v2/ui](http://localhosts:8668/v2/ui).
+
+When to query **CrateDB** and when **QuatumLeap**?. As a rule of thumb, prefer 
+working always with **QuantumLeap** for the following reasons:
+
+- Your experience will be closer to FIWARE NGSI APIs like Orion's.
+- Your application will not be tied to CrateDB's specifics nor QuantumLeap's 
+implementation details, which could change and break your app.
+- QuantumLeap can be easily extended to other backends and your app will get 
+compatibility for free.
+- If your deployment is distributed, you won't need to expose the ports of 
+your database to the outside.
+
+If your are sure your query is not supported by **QuantumLeap**, you may have 
+to end up querying **CrateDB**, however, please open an issue in 
+[QuantumLeap's repo](https://github.com/smartsdk/ngsi-timeseries-api/issues) 
+so the team is aware.
+
+### :three: Checking Data persistence
+
+#### Using QuantumLeap API
+
+To check you are persisting `luminosity` values of entities of type `Lamp`, 
+you can simply query the latest 5 records of that attr & type combination.
+Note the use of `Fiware-Service` and `Fiware-ServicePath` headers. These 
+are required only when data are pushed to orion using such headers 
+(in multitenancy scenarios). Failing to align these headers will result in 
+no data being returned.
+
+##### Request:
+
+```console
+curl -iX GET \
+  'http://localhost:8668/v2/types/Lamp/attrs/luminosity?lastN=5' \
+  -H 'Content-Type: application/json' \
+  -H 'Fiware-Service: openiot' \
+  -H 'Fiware-ServicePath: /'
+```
+
+##### Response:
+
+```json
+{
+  "data": {
+    "attrName": "luminosity",
+    "entities": [
+      {
+        "entityId": "Lamp:001",
+        "index": [
+          "2018-10-29T14:32:01.929000",
+          "2018-10-29T14:32:02.946000",
+          "2018-10-29T14:32:03.928000",
+          "2018-10-29T14:32:04.946000",
+          "2018-10-29T14:32:05.918000"
+        ],
+        "values": [
+          1894,
+          1897,
+          1903,
+          1907,
+          1988
+        ]
+      }
+    ],
+    "entityType": "Lamp"
+  }
+}
+```
 
-**QuantumLeap** does not currently offer any interfaces to query for the
-persisted data A good method to see if data is being persisted is to check to
-see if a `table_schema` has been created. This can be done by making a request
+Can you check the last 8 records of `count` for the `Motion` entities?
```

Comment:
I don't know can you? Don't leave me hanging like this. 😄 

Either supply the **Answer** or remove _Can you check the last 8 records of `count` for the `Motion` entities?_ - the tone here is incorrect and too familiar

Reply:
Sorry, never thought I'd be setting a wrong tone. Maybe I should've used *Could you*?. 

This kind of questions are very useful for didactic reasons IMO. You don't need the answer really. If the reader cannot respond, then he'll be encouraged to think about it before continuing.